main.cpp

```diff
@@ -84,42 +101,91 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 /*
  1)
  */
+bool buyACar(int price = 50000, bool used)
+{
+    ignoreUnused(price, used);
+    return {};
+}
 
 /*
  2)
  */
+int openFrontDoor(bool gotTheKey , int amountOfLocks)
+{
+    ignoreUnused(gotTheKey, amountOfLocks);
+    return {};
+}
 
 /*
  3)
  */
-
+float flyToBerlin(int ticketPrice, int flightNumber, int seats = 3)
+{
+    ignoreUnused(ticketPrice, flightNumber, seats);
+    return {};
+}
+
 /*
  4)
  */
-
+bool swimInSee(bool speedosOn = true, int sunScreen)
+{
+    ignoreUnused(speedosOn, sunScreen);
+    return {};
+}
+
 /*
  5)
  */
+void makeCoffee(int coffeeType, bool milk)
+{
+    ignoreUnused(coffeeType, milk);
+}
 
 /*
  6)
  */
+void washHair(int waterTemp = 38, bool blowDryer)
+{
+    ignoreUnused(coffeeType, blowDryer);
+}
 
 /*
  7)
  */
+int goToSleep(bool teethBrushed, bool tired)
+{
+    ignoreUnused(teethBrushed, tired);
+    return {};
+}
 
 /*
  8)
  */
+int cookDinner(int ingredients = 20, int cookingDuration)
+{
+    ignoreUnused(ingredients, cookingDuration);
+    return {};
+}
 
 /*
  9)
  */
+int writingFunctions(int dataType, int parameters)
```

Comment:
`writing` is not a verb.  
it is a present participle
`writeFunctions` is the correct function name to provide here
- [ ] revise